Presscotts_Mortar_Calculator_(no_envirement)_V0.2.py

```diff
@@ -19,215 +18,225 @@
-    print('='*90)
+    print('=' * 90)
     print('[OUTPUT]')
-  
+
     if len(posMortar) % 2 == 1:
         print('ERROR: lenth of gridmark is odd, renter gridmarks')
         main()
-    if len(posTarget)% 2 == 1:
-        print('ERROR: lenth of gridmark is odd, renter gridmarks')    
+    if len(posTarget) % 2 == 1:
+        print('ERROR: lenth of gridmark is odd, renter gridmarks')
         main()
     '''Determining X,Y coordinates and setup for distance'''
     posMortarXY = str(posMortar)
-    mUpper = ((len (posMortarXY))/2) 
-    mUpper = int(mUpper) 
+    mUpper = ((len(posMortarXY)) / 2)
+    mUpper = int(mUpper)
     xGridM = posMortarXY[0:mUpper:1]
     posMortarX = str(xGridM)
     posMortarX = int(posMortarX)
-    
-    mLower = ((len (posMortarXY )/2))
+
+    mLower = ((len(posMortarXY) / 2))
     mLower = int(mLower)
     yGridM = posMortarXY[mLower::1]
     posMortarY = str(yGridM)
     posMortarY = int(posMortarY)
-     
-     
+
     posTargetXY = str(posTarget)
-    tUpper = ((len (posTargetXY )/2)+1) 
+    tUpper = ((len(posTargetXY) / 2) + 1)
     tUpper = int(tUpper)
     xGridT = posTargetXY[0:mUpper:1]
     posTargetX = str(xGridT)
     posTargetX = int(posTargetX)
-    
-    tLower = ((len (posTargetXY )/2)) 
+
+    tLower = ((len(posTargetXY) / 2))
     tLower = int(tLower)
     yGridT = posTargetXY[mLower::1]
     posTargetY = str(yGridT)
     posTargetY = int(posTargetY)
-    
+
     #print(f'tartget x{posTargetX} target y{posTargetY} \n youX {posMortarX} yourY {posMortarY} ')
     ''' distance CALC'''
     dx = (posTargetX - posMortarX)
     #print(f'dx= {dx}')
     dy = (posTargetY - posMortarY)
     #print(f'dy= {dy}')
-    distance = math.sqrt(dx**2 + dy**2) 
-    
-    '''distance unit adjuestment'''    
-    
-    if len(posMortar) == 6: 
-        distance = 100*distance    
+    distance = math.sqrt(dx**2 + dy**2)
+    '''distance unit adjuestment'''
+
+    if len(posMortar) == 6:
+        distance = 100 * distance
     if len(posMortar) == 8:
-        distance = 10*distance
+        distance = 10 * distance
     if len(posMortar) == 10:
-        distance = distance    
+        distance = distance
     #print(distance)
     '''Determing azimuth in mills, and setting up for special cases where arctan is undefined'''
     '''change "if" statement to negative check and then assign ones or zeros to do a true false'''
-    azimuth=0
+    azimuth = 0
     #print (f' DX{dx}//DY{dy}')
     if dy == 0:
-        if  dx < 0:
+        if dx < 0:
             azimuth = 4800
-        if  dx > 0:
+        if dx > 0:
             azimuth = 1600
     #NE
-    if dx > 0 and dy > 0:   
+    if dx > 0 and dy > 0:
         azimuth = 17.77777777777777777778 * math.degrees(math.atan(dx / dy))
         #print('NE')
     #NW
-    if dx < 0 and dy > 0:   
-        azimuth = 6400 + 17.77777777777777777778 * math.degrees(math.atan(dx / dy))  
+    if dx < 0 and dy > 0:
+        azimuth = 6400 + 17.77777777777777777778 * math.degrees(

```

Comment:
Constants! Rather than repeating magic numbers, like `17.77777777777777777778`, prefer defining meaningful constants. Conventionally done in `SHOUTY_CASE`
i.e.
```
DEGREES_TO_MILS =  17.77777777777777777778 // Dunno what this actually converts lol. You'll have a better name I'm sure.
...
azimuth = 6400 + MILS_TO_DEGREES * math.degrees... 
```

Reply:
This could also be done as a function wrapper.